content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: 'Bias in User Research'
+Description: 'In user research, bias refers to prejudices or tendencies in researchers’ and participants’ ways of thinking that can lead to inaccurate or skewed results.'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In [user research](https://www.codecademy.com/resources/docs/uiux/user-research), __bias__ refers to prejudices or tendencies in researchers’ and participants’ ways of thinking that can lead to inaccurate or skewed results. 
+
+To obtain accurate, useful data, researchers must be aware of their own biases. Researchers can also work to prevent participants from displaying common biases by strategically designing their studies. When analyzing results, researchers should consider how bias may have impacted the data, especially if the data is self-reported. 
+
+### Acquiescence Bias 
+
+ __Acquiescence bias__ is the tendency to agree with the interviewer. In general, participants are more likely to agree than to disagree. Thus, the question should not imply that there is a "correct" answer. Instead of asking for agreement or disagreement to a predefined statement, researchers can ask users to explain their point of view.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+*“Would you agree that you are satisfied with the current interface?”* This question implies that the user is satisfied with the interface and could lead to acquiescence bias. <br>    
+
+*“How do you feel about the current interface?”* This phrasing is open-ended, so it does not lead participants to agree with a particular sentiment. </details> 
+
+### Confirmation Bias 
+
+ __Confirmation bias__ is the tendency for researchers to interpret data in a way that supports their opinions and beliefs. Researchers know the study’s goal, so they might make connections that aren’t explicit in the data set based on their opinions, beliefs, or expectations on how that goal should be met.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+This survey asked, “Which of the following emotions do you feel when at the airport? Please select the options below.”  
+
+![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](media/confirmation-bias-chart.png) 
+
+The team’s larger goal is to make a product that boosts positive feelings at the airport. Confirmation bias might mean looking at this data and only reporting findings based on the statistic “57.1 of our participants are excited when traveling at the airport.” While this is true, it ignores the other data and emotions, that 42.9% also feel anxious, confused, and frustrated.  </details> 
```

Comment:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

<p>This survey asked,</p> <q>Which of the following emotions do you feel when at the airport? Please select the options below.</q>  

![Bar Chart that displays the data from the survey question: Which of the following emotions do you feel at the airport?](media/confirmation-bias-chart.png) 

The team’s larger goal is to make a product that boosts positive feelings at the airport. Confirmation bias might mean looking at this data and only reporting findings based on the statistic “57.1 of our participants are excited when traveling at the airport.” While this is true, it ignores the other data and emotions, that 42.9% also feel anxious, confused, and frustrated.  </details> 
```
```